examples/GPU/example_learn_samples_multicoil.py

```diff
@@ -67,8 +67,9 @@ def __init__(self, inital_trajectory, n_coils, img_size=(256, 256)):
         )
         sample_points = inital_trajectory.reshape(-1, inital_trajectory.shape[-1])
         # A simple acquisition model simulated with a forward NUFFT operator. We dont need density compensation here.
+        # The trajectory is scaled by 2*pi for cufinufft backend.
         self.operator = get_operator("cufinufft", wrt_data=True, wrt_traj=True)(
-            sample_points,
+            sample_points*2*np.pi,
```

Comment:
This is a no-op, the trajectory is going to go through `proper_trajectory` for doing the scaling.